schemas/documents/shared/project.js

```diff
@@ -4,16 +4,31 @@ export default {
   type: 'document',
   fields: [
     {
-      title: 'Project name',
-      name: 'projectName',
+      title: 'Project id, uuid or slug',
+      name: 'projectId',
       type: 'string',
+      description: 'on-chain project id, off-chain uuid or slug',
       validation: Rule => Rule.required(),
     },
     {
-      title: 'Project id, uuid or slug',
-      name: 'projectId',
+      title: 'Project Pre-Financing',
+      name: 'projectPreFinancing',
+      type: 'projectPreFinancing',
```

Comment:
this is the main change here for the `project` document, which represents a project in sanity and is mapped through the on chain id, off chain uuid or slug
the rest of the changes in this file are just for re-ordering the fields